sdk/examples/wallet/07_send_native_tokens.rs

```diff
@@ -15,40 +20,51 @@ use iota_sdk::{
 };
 use primitive_types::U256;
 
+// The account alias used in this example
+const ACCOUNT_ALIAS: &str = "Alice";
```

Comment:
Isn't this really annoying if one wants to change it, because then it needs to be changed in all examples?
Maybe most examples should just get the first account by index 0 and not use the alias

Reply:
Changed.